daprdocs/data/components/state_stores/azure.yaml

```diff
@@ -1,6 +1,6 @@
 - component: Azure Blob Storage
   link: setup-azure-blobstorage
-  state: Stable
```

Comment:
If a component is already stable, it can stay that way. My question is more about how we can track the beta components that are stable candidates.

Reply:
Stable candidate has no official meaning - it's a process detail where we are announcing that the current Beta is considered complete and will be made stable in the future. Either we keep the current Beta designation and simply announce that the current Beta is a stable candidate (essentially the Beta release is a RC), or we get rid of this idea of stable candidate again.

Reply:
> If a component is already stable, it can stay that way. My question is more about how we can track the beta components that are stable candidates.

ok, so since blob storage certification tests were added in 1.8, we should mark it as a "stable candidate " in 1.8 and stable in 1.9 correct?